### PR TITLE
Integration Covalent historical API

### PR DESCRIPTION
**Pull request by castromaciel**
We made the first integration and add an example of how to use the Covalent API.

### CODE REVIEW

**Review by gigliagustin — CHANGES_REQUESTED** [review]
Hi Maciel,

To approve this PR, you have to review the comments that I leaved. Please, review it, make the changes and make the PR again.

File: src/services/getHistorical.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import axios from 'axios';
+import { key } from '../constants';
+
+// const date = new Date().now();
```
[the remark this answers is not in the document]
Comment (castromaciel) [reply src/services/getHistorical.js]: Mr Agucho, I've made another commit with the requested changes.

File: src/services/getHistorical.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+// const from = date; // Restar 7 dias;
```
[the remark this answers is not in the document]
Comment (castromaciel) [reply src/services/getHistorical.js]: Mr Agucho, I've made another commit with the requested changes.

**Review by gigliagustin — APPROVED** [review]
Thanks for making the changes dear Macielacho!

**Pull request merged**